src/pls/fs/stats.py

```diff
@@ -69,29 +70,29 @@ def get_size(st_size: int) -> str:
             return f"{magnitude}[dim]{unit}[/]"
     return f"{st_size}  [dim]B[/]"
 
+if platform != "win32":
+    def get_user(st_uid: int) -> str:
```

Comment:
It might be better to try the import inside the `get_user` and `get_group` functions, catch the `ModuleNotFoundException` which is raised on Windows, and return None in that case. It will save a lot of `if platform != "win32"` checks.